util.py

```diff
@@ -372,3 +372,15 @@ def get_g2p_python(g2p_python: tk.Path) -> tk.Path:
 def get_subword_nmt_repo(subword_nmt_repo: tk.Path) -> tk.Path:
     """gets the path to the root folder of subword-nmt repo"""
     return get_executable_path(subword_nmt_repo, "SUBWORD_NMT_PATH")
+
+
+def get_ngram_count_exe(count_exe: tk.Path):
+    return get_executable_path(count_exe, None, shutil.which("ngram-count"))
```

Comment:
Do we really want to support this? I would prefer no default, so that you always have to give this explicitly. We anyway do not know yet if it would be allowed for us to include SRILM directly in Apptainer images.

Reply:
so if the `str` for the `shutil.which` is not found `None` is returned. the fould lead to an assertion error. I think this is ok. but I do not have a strong opinion on this.. @Atticus1806 what do you say?

Reply:
When  I think about it I don't really think we should have "hidden defaults" for executables. 99% of the time you want to have this set anyways right? And you get the benefit of 1% "nice behavior" with the risk of having unintended behavior in a case where you just forgot. So I would not mind crashing here.
We also did similar things for `returnn_exe` etc. right? Where we disallowed pulling from `gs` which I would say is similar to this.